README.md

```diff
@@ -42,22 +42,17 @@ const configuration = new Configuration({
 const openai = new OpenAIApi(configuration);
 
 const completion = await openai.createCompletion({
-  // the model will be ignored and the configured model from the project will be used
   model: "text-davinci-003",
-  // prompt: "Hello World!", // prompt is not needed anymore
-  // set this to you project id
+  prompt: "Hello World, my name is Alice!",
+  // you need to add your project id
```

Comment:
Perhaps add why that's necessary? We will likely need some docs about this soonish.